PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/BaseSequence.java

```diff
@@ -0,0 +1,170 @@
+package nl.tudelft.pl2016gr2.model;
+
+import nl.tudelft.pl2016gr2.thirdparty.testing.utility.TestId;
+
+/**
+ * This is a container class for the bases of a genome. It efficiently encodes the ATCG and N
```

Comment:
`This is a container class` comes over as if it is a data class. All classes are 'containers' for some sort of abstraction. Consider changing this to something along the lines of "Represents a DNA sequence."